cli-toolchain/graphql.md

```diff
@@ -2379,6 +2379,157 @@ Here is a complete list of searchable operations per GraphQL type supported as o
 | Boolean | `eq`, `ne`      |
 
 
+
+### @predictions
+
+The `@predictions` directive allows one to query an orchestration of AI/ML services such as Amazon Rekognition, Amazon Translate, and/or Amazon Polly.
```

Comment:
Would it make sense to link back to [the docs of how to add predictions](https://aws-amplify.github.io/docs/js/predictions#predictions) in some way?

Reply:
I suppose we could to outline the services being used though implementation is different.